tournamentcontrol/competition/tests/test_utils.py

```diff
@@ -124,3 +124,98 @@ def test_sum_dict_integer(self):
 
     def test_revpow(self):
         self.assertEqual(utils.revpow(2**4, 2), 4)
+
+
+class StageGroupPositionTests(TestCase):
+    """
+    Tests for the stage_group_position function.
+    """
+
+    def test_stage_group_position_with_no_pools_raises_index_error(self):
+        """
+        Test that stage_group_position raises IndexError when trying to access
+        a pool that doesn't exist on a stage.
+        """
+        from tournamentcontrol.competition.tests.factories import (
```

Comment:
Imports should be added at the top of the file. There is no reason for this inline importing.